index.html

```diff
@@ -343,8 +343,11 @@
           <li>If the <a>browsing context</a> of the script calling the
           constructor is a <a>nested browsing context</a> whose origin is
```

Comment:
I think this PR should be blocked on fixing these issues:
https://github.com/w3c/browser-payment-api/issues/324
https://github.com/w3c/browser-payment-api/issues/332

The whole paragraph doesn't any make sense right now and does not seem actually secure.